app/Services/JobStatusTransitions.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\User;
+
+class JobStatusTransitions
+{
+    protected $transition_graph = [
+        'states' => [
```

Comment:
I think it makes sense to store these states and transitions in the database, then we could make actual relationships between things like owners and user roles, and provide localized text for names and/or descriptions.

I used a `key` and `name` field convention for storing the text key and localized name respectively on the User Role edits I made.